bin/tsc

```diff
@@ -1,2 +1,13 @@
 #!/usr/bin/env node
-require('../lib/tsc.js')
+var resolve = require('resolve').sync;
+var path = require('path');
+var basedir = process.cwd();
```

Comment:
@mprobst in clang-format you use the last arg as a hint where the node_modules directory can be located, then fallback to process.cwd(). I suppose this allows you to run from anywhere, do you think users do that?

There may be a different problem here - the `-p` flag. If the user is in their home directory, I think they could run `tsc -p path/to/my/project` - so maybe we should use the value of `-p` as a hint?